File: API/Features/Billing/Prices/ViewModels/PriceListVM.cs
using System;
using API.Infrastructure.Classes;

namespace API.Features.Billing.Prices {

    public class PriceListVM {

        public Guid Id { get; set; }
        public SimpleEntity Customer { get; set; }
        public SimpleEntity Destination { get; set; }
        public SimpleEntity Port { get; set; }
        public string From { get; set; }
        public string To { get; set; }
        public decimal AdultsWithTransfer { get; set; }
        public decimal AdultsWithoutTransfer { get; set; }
        public decimal KidsWithTransfer { get; set; }
        public decimal KidsWithoutTransfer { get; set; }

    }

}

File: FrontEnd/src/app/features/billing/prices/classes/view-models/price-list-vm.ts
import { SimpleEntity } from './../../../../../shared/classes/simple-entity'

export interface PriceListVM {

    id: number
    customer: SimpleEntity
    destination: SimpleEntity
    port: SimpleEntity
    from: string
    formattedFrom: string
    to: string
    formattedTo: string
    adultsWithTransfer: number
    adultsWithoutTransfer: number
    kidsWithTransfer: number
    kidsWithoutTransfer: number

}
